instrumentation/instaawssdk/invoke.go

```diff
@@ -0,0 +1,66 @@
+package instaawssdk
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+
+	otlog "github.com/opentracing/opentracing-go/log"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	instana "github.com/instana/go-sensor"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+func StartInvokeSpan(req *request.Request, sensor *instana.Sensor) {
+	tags := opentracing.Tags{}
+	if ii, ok := req.Params.(*lambda.InvokeInput); ok {
```

Comment:
Can we do this after the span has been started to not to create an additional `opentracing.Tags` map? We'd also skip this entirely if the caller is not traced.

Reply:
Good idea 👍